CUDADataFormats/SiPixelDigi/interface/SiPixelDigisCUDASOAView.h

```diff
@@ -0,0 +1,83 @@
+#ifndef CUDADataFormats_SiPixelDigi_interface_SiPixelDigisCUDASOAView_h
+#define CUDADataFormats_SiPixelDigi_interface_SiPixelDigisCUDASOAView_h
+
+#include <cuda_runtime.h>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+
+#include <cstdint>
+
+auto roundFor128ByteAlignment = [](int numFedWords) {
+  int mul = 128 / sizeof(uint16_t);
+  return ((numFedWords + mul - 1) / mul) * mul;
+};
```

Comment:
This would be better as a `static` member function of `SiPixelDigisCUDASOAView` (or generalized and put in e.g. `HeterogeneousCore/CUDAUtilities` as `inline` function).

Reply:
why this is a lambda instead of a normal function ?